app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -74,23 +81,25 @@
       app:layout_constraintTop_toBottomOf="@+id/error_text"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintEnd_toEndOf="parent"/>
+
     <TextView
       android:id="@+id/forgot_pin"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginTop="10dp"
+      android:layout_height="48dp"
+      android:layout_marginTop="12dp"
+      android:fontFamily="sans-serif"
+      android:lineSpacingExtra="20sp"
       android:text="@string/pin_password_forgot_pin"
       android:textAllCaps="true"
       android:textColor="@color/colorPrimary"
```

Comment:
primaryColor in the application is `#00645C` and the color in the mocks is `#009688` , but I think it was because of the fact that we had updated the mocks color in between. Maybe trying to clear this with Chantel, which color do we need here.

Reply:
@mschanteltc can you please clear, which color we are supposed to keep for the text `I forgot pin`

Reply:
It would be #00645C, but against a #F0FFFF background, it would pass WCAG AA but not WCAG AAA. It is 0.14 shy off from the minimum requirement, but it is possible we can make this an exception?

Reply:
> > > Please confirm regarding the border with @mschanteltc because in mock there are not borders to the pinview. Also please use proper eye icons as per mock.
> > 
> > 
> > @mschanteltc can you please provide me icons to show and hide password.
> 
> Yes, it's available in the Android Image drive under the folder "Profile Chooser."

There is slight confusion in the show/hide icon along with the text. As per the [mock](https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/686c27d1-f07b-4680-95da-7767d71558a4/PC-AP-Empty-/) hide_eye_icon is been used with show text. But in our implementation it is other way. Which one is correct in this.
![Screenshot_20200110-111234](https://user-images.githubusercontent.com/16301028/72128905-63c42300-339a-11ea-85a9-73cb1afa340f.png)
![Screenshot_20200110-111244](https://user-images.githubusercontent.com/16301028/72128911-6757aa00-339a-11ea-9304-9c3d641c4e7d.png)

Reply:
Thanks @mschanteltc for clarification I will be using #00645C for textcolor against a #F0FFFF background.